src/collections/match/helpers/match-finder-2/match-meeting-info.spec.ts

```diff
@@ -0,0 +1,134 @@
+const kristine = createFakeMatchableUser(
+  "kristine",
+  "Mykhetens leksikon",
+  "Spirituell Spire"
+);
+const elRi = createFakeMatchableUser("elRi", "Spirituell Spire");
+
+/**
+ * Prints data about how many matches each location has at each timeslot
+ * @param matches matches that have been assigned a location and a timeslot
+ */
+function printLocationMetrics(matches: MatchWithMeetingInfo[]) {
+  const aggregatedLocationInfo = matches
+    .sort((a, b) => (a.meetingInfo.date > b.meetingInfo.date ? 1 : -1))
+    .reduce((acc, match) => {
+      const { location, date } = match.meetingInfo;
+      return {
+        ...acc,
+        [location]: {
+          ...acc[location],
+          count:
+            acc[location]?.count === undefined ? 1 : acc[location].count + 1,
+          [String(date)]:
+            acc[location]?.[String(date)] === undefined
+              ? 1
+              : acc[location][String(date)] + 1,
+        },
+      };
+    }, {});
+  console.log(aggregatedLocationInfo);
+}
+
+describe("Simple Matches", () => {
+  it("should be able to assign non overlapping time a simple match setup", () => {
+    const matchFinder = new MatchFinder([audun, kristine], [siri, elRi]);
+    const matches = matchFinder.generateMatches();
+    const standLocation = "Resepsjonen";
+    // Fails if match verification throws an error
+    assignMeetingInfoToMatches(
+      matches,
+      standLocation,
+      [{ name: "Sal 1" }, { name: "Sal 2" }],
+      new Date("2023-02-02T12:00:00+0100")
+    );
+  });
+
+  it("should not be able to be more matches than the location limit at a given time", () => {
+    const senders = createUserGroup("sender", 10, "A", "B", "C");
+    const receivers = createUserGroup("receiver", 10, "A", "B", "C");
+    const matchFinder = new MatchFinder(senders, receivers);
+    const matches = matchFinder.generateMatches();
+    const standLocation = "Resepsjonen";
+    const simultaneousMatchLimit = 2;
+    // Fails if match verification throws an error
+    const updatedMatches = assignMeetingInfoToMatches(
+      matches,
+      standLocation,
+      [{ name: "Sal 1", simultaneousMatchLimit }, { name: "Sal 2" }],
+      new Date("2023-02-02T12:00:00+0100")
+    );
+    const meetingTimes = updatedMatches
+      .filter((match) => match.meetingInfo.location === "Sal 1")
+      .map((match) => match.meetingInfo.date);
+
+    const distinctMeetingTimes = meetingTimes.reduce(
+      (acc, date) =>
+        acc.some((existingDate) => existingDate.getTime() === date.getTime())
+          ? acc
+          : [...acc, date],
+      [] as Date[]
+    );
+
+    for (const distinctMeetingTime of distinctMeetingTimes) {
+      const simultaneousMatches = meetingTimes.reduce(
+        (acc, next) =>
+          acc + (next.getTime() === distinctMeetingTime.getTime() ? 1 : 0),
+        0
+      );
+      expect(simultaneousMatches <= simultaneousMatchLimit).to.be.true;
```

Comment:
```suggestion
      expect(simultaneousMatches).to.be.lessThanOrEqual(simultaneousMatchLimit);
```
Better failure message